eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/TopicHandler.java

```diff
@@ -85,8 +86,8 @@ void get(HttpExchange httpExchange) throws IOException {
         try {
             List<TopicProperties> topicList = admin.getTopic();
             String result = JsonUtils.toJson(topicList);
-            httpExchange.sendResponseHeaders(200, result.getBytes().length);
-            out.write(result.getBytes());
+            httpExchange.sendResponseHeaders(200, result.getBytes(StandardCharsets.UTF_8).length);
```

Comment:
I remember that the public module should have the definition of charsets, which you can use directly.